pyflakes/checker.py

```diff
@@ -1546,7 +1493,7 @@ def SUBSCRIPT(self, node):
 
     def _handle_string_dot_format(self, node):
         try:
-            placeholders = tuple(parse_format_string(node.func.value.s))
+            placeholders = tuple(string.Formatter().parse(node.func.value.s))
```

Comment:
don't inline this, it's intentionally module scoped because it is expensive to construct a `Formatter()` repeatedly